Kernel/FileSystem/FATFS/Inode.cpp

```diff
@@ -31,10 +31,10 @@ FATInode::FATInode(FATFS& fs, FATEntry entry, NonnullOwnPtr<KString> filename)
         .uid = 0,
         .gid = 0,
         .link_count = 0,
-        .atime = fat_date_time(m_entry.last_accessed_date, { 0 }),
-        .ctime = fat_date_time(m_entry.creation_date, m_entry.creation_time),
-        .mtime = fat_date_time(m_entry.modification_date, m_entry.modification_time),
-        .dtime = 0,
+        .atime = Time::from_timespec({ fat_date_time(m_entry.last_accessed_date, { 0 }), 0 }),
```

Comment:
A simpler way to solve this would be to remove `.to_seconds()` in fat_date_time() and make it return a Time instead of time_t. Should be safe as these are all callsites for that function

Reply:
True, that's nicer! Let's do that.